src/python/WMComponent/AnalyticsDataCollector/Plugins/Tier0Plugin.py

```diff
@@ -180,15 +183,14 @@ def __call__(self, requestDocs, localSummaryCouchDB, centralRequestCouchDB):
                 workflowStatus = self.determineCurrentStatus(workflowName, workflowType, completedTaskList)
                 if workflowStatus is not None:
                     centralRequestCouchDB.updateRequestStatus(workflowName, workflowStatus)
-
+            except Tier0PluginError as t0ex:
```

Comment:
Good one!

Reply:
Took me a minute to understand what pylint was telling me which is that the 2nd exception was a subclass of the first, ergo never gets run.